smarts/core/smarts.py

```diff
@@ -850,12 +850,16 @@ def __del__(self):
             self.destroy()
         except (TypeError, AttributeError) as e:
             # This is a print statement because the logging module may be deleted at program exit.
-            raise SMARTSDestroyedError(
-                "ERROR: A SMARTS instance may have been deleted by gc before a call to destroy."
-                " Please explicitly call `del obj` or `SMARTS.destroy()` to make this error"
-                " go away.",
-                e,
-            )
+            try:
+                exception = SMARTSDestroyedError(
```

Comment:
Why is this in a try/catch block?

Reply:
This is an odd issue. When delete is called the module attributes are sometimes already deleted. In this case the `SMARTSDestroyedError` type is deleted.